express-get-json/index.js

```diff
@@ -0,0 +1,36 @@
+const express = require('express');
+const app = express();
+const array = [];
```

Comment:
This should be in the callback function for your `/api/grades` route so that a new array is created when someone makes a request to the server.  How it is now the array will get bigger and bigger with each request.